site2/docs/admin-api-namespaces.md

```diff
@@ -288,9 +288,9 @@ admin.namespaces().getNamespaceReplicationClusters(namespace)
 
 Backlog quota helps the broker to restrict bandwidth/storage of a namespace once it reaches a certain threshold limit. Admin can set the limit and take corresponding action after the limit is reached.
 
-  1.  producer_request_hold: broker holds but not persists produce request payload
+  1.  producer_request_hold: the producer holds the message and retries until client configuration `operationTimeoutMs` is exceeded
```

Comment:
It should be until client configuration `sendTimeoutMs` is exceeded.
The `operationTimeoutMs` is for producer creation or consumer creation.

Reply:
Addressed. Good catch. `operationTimeoutMs` applies for producer creation but `sendTimeoutMs` applies for send requests.

Reply:
Addressed.